pjmedia/include/pjmedia-codec/types.h

```diff
@@ -126,7 +126,11 @@ enum pjmedia_video_pt
      PJMEDIA_RTP_PT_H264_RSV4,
 
      PJMEDIA_RTP_PT_VP8,
+     PJMEDIA_RTP_PT_VP8_RSV1,
+     PJMEDIA_RTP_PT_VP8_RSV2,
      PJMEDIA_RTP_PT_VP9,
+     PJMEDIA_RTP_PT_VP9_RSV1,
+     PJMEDIA_RTP_PT_VP9_RSV2,
```

Comment:
A short comment for each PT (non-doxygen) about the 'owner' of the PT slot perhaps help tracking in the future, e.g:
```PJMEDIA_RTP_PT_H264, /* used by OpenH264 */```